src/main/scala/com/moesif/filter/MoesifApiFilter.scala

```diff
@@ -195,7 +154,7 @@ class MoesifApiFilter @Inject()(config: MoesifApiFilterConfig)(implicit mat: Mat
 
   def sendEvent(eventModel: EventModel): Unit = synchronized {
       val randomPercentage = Math.random * 100
-      val sampleRateToUse = getSampleRateToUse(eventModel.getUserId, eventModel.getCompanyId)
+      val sampleRateToUse = moesifApi.getSampleRateToUse(eventModel)
```

Comment:
You might want to make this line  val eventModelMasked = advancedConfig.maskContent(eventModel) down here.

Reply:
Make sense, changed